vegafusion-common/src/data/table.rs

```diff
@@ -271,6 +272,13 @@ impl VegaFusionTable {
         }
     }
 
+    #[cfg(feature = "pyarrow")]
+    pub fn from_arrow_c_stream(table: &Bound<PyAny>) -> PyResult<Self> {
+        let pytable = pyo3_arrow::PyTable::from_py_object_bound(table.as_borrowed())?;
+        let (batches, schema) = pytable.into_inner();
+        Ok(VegaFusionTable::try_new(schema, batches)?)
```

Comment:
I never know whether `batches, schema` or `schema, batches` is a better ordering 🥲